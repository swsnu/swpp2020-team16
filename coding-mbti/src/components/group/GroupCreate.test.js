describe('GroupCreate', () => {
    it('render', () => {
        expect(1).toEqual(1);
    });
});
