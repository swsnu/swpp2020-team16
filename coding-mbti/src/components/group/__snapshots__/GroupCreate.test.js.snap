// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GroupCreate no Manager 1`] = `ReactWrapper {}`;

exports[`GroupCreate render 1`] = `ReactWrapper {}`;
